apps/prairielearn/python/prairielearn/grading_utils.py

```diff
@@ -166,14 +172,37 @@ def grade_answer_parameterized(
     weight: int = 1,
 ) -> None:
     """
-    Grade question question_name. grade_function should take in a single parameter
-    (which will be the submitted answer) and return a 2-tuple:
-        - The first element of the 2-tuple should either be:
-            - a boolean indicating whether the question should be marked correct
-            - a partial score between 0 and 1, inclusive
-        - The second element of the 2-tuple should either be:
-            - a string containing feedback
-            - None, if there is no feedback (usually this should only occur if the answer is correct)
+    Grade question `question_name` using the provided `grade_function`.
```

Comment:
We don't call these "questions" anywhere else, not even in the function name itself. Can we call these "answers" and maybe rename the `question_name` argument to just `name`? Ditto with the example below (`"question_name"` -> something else).

Reply:
Changed in [6afd3f2](https://github.com/PrairieLearn/PrairieLearn/pull/11761/commits/6afd3f220e9b7faa06cae6f989c51bc6b6916b4f).